axelar-chains-config/info/mainnet.json

```diff
@@ -1367,7 +1370,10 @@
         "gasPrice": 10000000000
       },
       "finality": "finalized",
-      "approxFinalityWaitTime": 25
+      "approxFinalityWaitTime": 25,
+      "onchainGasEstimate": {
+        "gasEstimationType": 1
```

Comment:
add L1 chain name here

Reply:
@milapsheth should we also make an api call for the L1 chain for the right base fee?
Right now blob base fee is calculated via source/destination price ratio, not L1.

Reply:
Disregard most L2s have Ether as gas token.